### PR TITLE
16525 Fixed unit test errors

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#16525

*Description of changes:*
- app version = 6.5.1
- fixed feature flag mocking
- enabled tests
- fixed lint issue
- mocked console.warn in a test suite to hide expansion panel warnings

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: tests/unit/allowable-actions-mixin.spec.ts [hunk 1]
```diff
@@ -15,6 +15,13 @@ const vuetify = new Vuetify({})
 setActivePinia(createPinia())
 const businessStore = useBusinessStore()
 
+// mock the entire module
+// it's the only way to override any exported function
+jest.mock('@/utils/feature-flags', () => {
```
Comment (JazzarKarim) [inline tests/unit/allowable-actions-mixin.spec.ts:20]: Oh OK. So, this'll provide a way to actually use the GetFeatureFlag function in this test. I'll be keeping this in mind.

**Review by JazzarKarim — APPROVED** [review]
Thank you so so much for helping out with fixing the test Sev 😃

File: tests/unit/allowable-actions-mixin.spec.ts (hunk 1 above)
Comment (severinbeauvais) [reply tests/unit/allowable-actions-mixin.spec.ts:20]: The old code worked fine before the package updates... 🤷‍♂️

File: src/components/common/Directors.vue [hunk 2]
```diff
@@ -591,15 +591,11 @@ export default class Directors extends Mixins(CommonMixin, DateMixin, DirectorMi
...
-    deliveryAddress: undefined,
-    appointmentDate: '',
-    cessationDate: '',
-    actions: []
-  }
+  /**
+   * Define dir so it's reactive and can be referenced in the template's for loop.
+   * Not sure why this is needed in this case but not others.
+   */
+  dir = {} // no type needed
```
Comment (severinbeauvais) [inline src/components/common/Directors.vue:598]: I couldn't figure this one out in a reasonable time so just did this.

File: tests/unit/LedgerConsentContinuationOut.spec.ts [hunk 3]
```diff
@@ -4,6 +4,10 @@ import { mount } from '@vue/test-utils'
 import ConsentContinuationOut from '@/components/Dashboard/FilingHistoryList/filings/ConsentContinuationOut.vue'
 import { createPinia, setActivePinia } from 'pinia'
 
+// mock the console.warn function to hide "[Vuetify] The v-expansion-panel component must be used inside a
+// v-expansion-panels"
+console.warn = jest.fn()
+
```
Comment (severinbeauvais) [inline tests/unit/LedgerConsentContinuationOut.spec.ts:10]: As mentioned earlier, I spent some time trying to fix this using `parentComponent` but bumped into other issues. Not worth a lot more time to fix this (IMHO).

**Pull request merged**